code/game/machinery/doors/airlock_types.dm

```diff
@@ -310,213 +310,173 @@
 /obj/machinery/door/airlock/almayer/generic/corporate
 	name = "Corporate Liaison's Quarters"
 	icon = 'icons/obj/doors/almayer/personaldoor.dmi'
-	req_access_txt = "200"
+	req_access = list(ACCESS_NT_CORPORATE)
 
 /obj/machinery/door/airlock/almayer/marine
 	name = "\improper Airlock"
 	icon = 'icons/obj/doors/almayer/prepdoor.dmi'
-	opacity = 0
-	glass = 1
+	opacity = FALSE
+	glass = TRUE
 
 /obj/machinery/door/airlock/almayer/marine/requisitions
 	name = "\improper Requisitions Bay"
 	icon = 'icons/obj/doors/almayer/prepdoor.dmi'
-	req_access_txt = "0"
-	req_one_access_txt =  "2;21"
-	opacity = 0
-	glass = 1
+	req_access = list()
+	req_one_access = list(ACCESS_MARINE_LOGISTICS, ACCESS_MARINE_CARGO)
+	opacity = FALSE
+	glass = TRUE
 
 /obj/machinery/door/airlock/almayer/marine/alpha
 	name = "\improper Alpha Squad Preparations"
 	icon = 'icons/obj/doors/almayer/prepdoor_alpha.dmi'
-	req_access_txt = "9"
-	req_one_access_txt =  "2;15"
-	opacity = 0
-	glass = 1
+	req_access = list(ACCESS_MARINE_PREP)
+	req_one_access = list(ACCESS_MARINE_LOGISTICS, ACCESS_MARINE_ALPHA)
+	opacity = FALSE
+	glass = TRUE
 
 /obj/machinery/door/airlock/almayer/marine/alpha/sl
 	name = "\improper Alpha Squad Leader Preparations"
-	req_access_txt ="12;15"
-	req_one_access_txt =  "0"
-	dir = 2
-	opacity = 0
-	glass = 1
+	req_access = list(ACCESS_MARINE_LEADER, ACCESS_MARINE_ALPHA)
+	req_one_access =  list()
+	dir = SOUTH
```

Comment:
Could make it even more compact by moving the dir to the parent on these

Reply:
Right, I'll get on it quickly.

Reply:
Done